assets/js/modules/analytics/components/common/TrackingExclusionSwitches.js

```diff
@@ -81,29 +78,26 @@ export default function TrackingExclusionSwitches() {
 			<legend className="googlesitekit-setup-module__text">
 				{ __( 'Exclude from Analytics', 'google-site-kit' ) }
 			</legend>
-			<Grid>
-				<Row>
-					<Cell lgSize={ 6 } mdSize={ 4 } smSize={ 4 }>
+			<div>
```

Comment:
@johnPhillips I think it would be better to add a new class to the container div (let's say `googlesitekit-settings-module__inline-items`) and use CSS flex styles to make children elements inline instead of introducing a new ad-hoc class for a very narrow use case. In other words, let's have a reusable CSS class that we can use in the future for other elements as well.

Reply:
Thanks @eugene-manuilov. I have added two more generic classes and an atomic rule for the top margin. Let me know what you think?

Reply:
Sounds good to me. Thanks @johnPhillips.